api/app.js

```diff
@@ -2,42 +2,134 @@ var createError = require('http-errors');
 var express = require('express');
 var path = require('path');
 var cookieParser = require('cookie-parser');
+var bodyParser = require("body-parser");
 var logger = require('morgan');
 var cors = require('cors');
 require('dotenv').config();
+var router = express.Router();
+var port = 9000;
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
 var coursesRouter = require('./routes/courses');
 var reviewsRouter = require('./routes/reviews');
+var authRouter = require('./routes/auth-routes');
+
+const cookieSession = require("cookie-session");
+const session = require('express-session');
+const passport = require('passport');
+const passportLocalMongoose = require('passport-local-mongoose');
+const googleSetup = require("./google-setup");
+const findOrCreate = require('mongoose-findorcreate');
 
 var app = express();
 
+app.use(
+  cookieSession({
+    name: "session",
+    keys: "thisappisawesome",
```

Comment:
It should be `keys: ["thisappisawesome"]`.  This was the one that was causing the error.